piazza_api/network.py

```diff
@@ -173,6 +173,26 @@ def create_instructor_answer(self, post, content, revision, anonymous=False):
         }
         return self._rpc.content_instructor_answer(params)
 
+    def mark_as_duplicate(self, duplicated_cid, master_cid, msg=''):
+        """Mark the post at `duplicated_cid` as a duplicate of `master_cid`
```

Comment:
I get what you're saying, but the existing style is incorrect and will not render properly with Sphinx, so it should be changed. But I guess that change can be done at once, and it's fine for now.